installation/configuration.md

```diff
@@ -120,77 +141,32 @@ rewrite ^/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2;
 rewrite ^/([^/\.]+)/([^/\.]+)/([^/\.]+)/?$ /index.php?act=$1.$2&id=$3;
 ```
 
-This and by desabling the 
+## Theme Configuration
+
+Leantime allows you to white label the entire system by changing the color scheme of your site, set your own logo, and also set primary and secondary color. This can be easily done by updating the `config/.env` file:
 
-## Language
-To change the default language of your installation just update   
 ```php
-public $language = 'en-US'; 
+LEAN_LOGO_PATH='/dist/images/logo.svg'           # Default logo path, can be changed later
+LEAN_PRINT_LOGO_URL='/dist/images/logo.jpg'      # Default logo URL use for printing (must be jpg or png format)
+LEAN_DEFAULT_THEME='default'                     # Default theme
+LEAN_PRIMARY_COLOR='#1b75bb'                     # Primary Theme color
+LEAN_SECONDARY_COLOR='#81B1A8'                   # Secondary Theme Color
 ```
-to a language string available. You can see all available languages in `resources\languages\languagelist.ini`
 
-## Timezone
-To change the default timezone of your installation, just update
+These settings can also be changed within Leantime under Account->Theme.
+
+## Sitename, Language and Timezone
+
+Configure sitename, your default language and timezone by updating the default settings section in `config/.env`:
+
```

Comment:
@marcelfolaron 

What's the blend here between when this should be called out in the code vs how you can also set it in system? i.e. should a link be placed here for how to set these things through the UI?

Reply:
I think we can skip that for now. I don't want to have to maintain that depending on what we add/remove from ui